scheduler/cloudformation/cloudformation.go

```diff
@@ -911,17 +935,30 @@ func (m *Scheduler) Run(ctx context.Context, app *scheduler.App, process *schedu
 		return errors.New("provided template can't generate a container definition for this process")
 	}
 
+	containerDefinition := t.ContainerDefinition(app, process)
+	if attached {
+		if containerDefinition.DockerLabels == nil {
+			containerDefinition.DockerLabels = make(map[string]*string)
+		}
+		// NOTE: Currently, this depends on a patched version of the
+		// Amazon ECS Container Agent, since the official agent doesn't
+		// provide a method to pass these down to the `CreateContainer`
+		// call.
+		containerDefinition.DockerLabels["docker.config.Tty"] = aws.String("true")
```

Comment:
The `Tty` flag allocates a psuedo TTY for the container, and `OpenStdin` opens the stdin stream.

Without `OpenStdin`, running something like `emp run bash` would just exit as soon as bash hit the prompt. Without `Tty`, I think most commands that expect a TTY/terminal wouldn't work (If I try `docker run -i ubuntu bash` (missing `-t` flag), my terminal just hangs).